src/service/user.service.ts

```diff
@@ -1,15 +1,46 @@
 import { Service } from 'typedi';
 import { InjectRepository } from 'typeorm-typedi-extensions';
 
-import { User } from '../database/entity';
 import { UserRepository } from '../database/repository';
+import { encryption } from '../utils/password.util';
 
 @Service()
 class UserService {
   constructor(@InjectRepository() private readonly userRepository: UserRepository) {}
 
-  public async userEmailExist(user: User) {
-    return !!this.userRepository.findByEmail(user.email);
+  public async isExistEmail({ email }: { email: string }) {
```

Comment:
이메일이 중복인지 아닌지 검사하는거라면 인자 한개로 받아도 될거 같아!
그리고 isExistingEmail 명이 더 자연스러운거 같아!
```suggestion
  public async isExistingEmail(email: string) {
```